src/components/GameView/FourDialog.vue

```diff
@@ -64,7 +67,7 @@ export default {
       },
     },
     hand() {
-      return this.$store.state.game.players[this.$store.state.game.myPNum].hand;
+      return this.gameStore.players[this.gameStore.myPNum].hand;
```

Comment:
We can use the player getter to dry this up 
```suggestion
      return this.gameStore.player.hand;
```